### ISSUE TITLE
The last commit probably miss a file

### ISSUE THREAD

**Issue by didiemme** [maturk/dn-splatter#69]
DepthToNormal class can't be find in any file in the repo.
Previous commit seems to work.

```
Traceback (most recent call last):
  File "/root/miniconda3/envs/dn_splat/bin/gs-mesh", line 5, in <module>
    from dn_splatter.export_mesh import entrypoint
  File "/workspace/dn-splatter/dn_splatter/__init__.py", line 3, in <module>
    from .data.mushroom_dataparser import MushroomDataParserSpecification
  File "/workspace/dn-splatter/dn_splatter/data/mushroom_dataparser.py", line 31, in <module>
    from dn_splatter.scripts.depth_to_normal import DepthToNormal
ModuleNotFoundError: No module named 'dn_splatter.scripts.depth_to_normal'
```

**Comment by maturk** [comment]
I think you are right @didiemme, thank's a lot for pointing this out! I will fix it...

**Comment by maturk** [comment]
should be fixed in #70 . Please re-open the issue if you find any more bugs like this! Thanks for your help!
